chart/README.md

```diff
@@ -29,7 +29,7 @@ If you're unfamiliar with Helm it would behoove you to peep around the documenta
 If you previously installed csi-driver with this command:
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/hetznercloud/csi-driver/latest/deploy/kubernetes/hcloud-csi.yml
+kubectl apply -f https://raw.githubusercontent.com/hetznercloud/csi-driver/v2.4.0/deploy/kubernetes/hcloud-csi.yml
```

Comment:
Shouldn't the version also be applied in the command below ?

Reply:
Fixed.